src/components/DatasetSettings.vue

```diff
@@ -46,36 +46,68 @@
 
-        <ListValues
-            label="Languages"
-            :items="context.allLanguages"
-            :single-item="context.language"
-            input-placeholder="LanguageCode"
-            new-item-placeholder="New Language"/>
-
-        <ListValues
-            label="Currencies"
-            :items="context.allCurrencies"
-            :single-item="context.currencyCode"
-            input-placeholder="CurrencyCode"
-            new-item-placeholder="New Currency"/>
-        
+        <ListValues label="Languages"
+                    :items="context.allLanguages"
+                    :single-item="context.language"
+                    input-placeholder="LanguageCode"
+                    new-item-placeholder="New Language"/>
+
+        <ListValues label="Currencies"
+                    :items="context.allCurrencies"
+                    :single-item="context.currencyCode"
+                    input-placeholder="CurrencyCode"
+                    new-item-placeholder="New Currency"/>
+
         <label class="text-sm block mt-6">Server url</label>
         <input v-model="context.serverUrl" type="text" placeholder="Server Url">
 
         <hr class="my-8">
-        
-        <label class="flex mt-2 items-center">
-            <input v-model="context.allowThirdLevelCategories" class="accent-brand-500 mr-3 h-5 w-5" type="checkbox">
-            Third level categories</label>
-
-        <p class="text-gray-400">
-            When third level categories are enabled, category links will also be rendered on the PLP's of categories with parents.<br>
-            This will allow users to navigate to a PLP of third level category.
-        </p>
-        
+
+        <h2 class="text-2xl mb-6">
+            Features
+        </h2>
+
+        <div class="pl-8 flex flex-col gap-4">
+            <div>
+                <label class="flex mt-2 items-center">
+                    <input v-model="context.allowThirdLevelCategories"
+                           class="accent-brand-500 mr-3 h-5 w-5"
+                           type="checkbox">
+                    Third level categories</label>
+
+                <p class="text-gray-400">
+                    When third level categories are enabled, category links will also be rendered on the PLP's of
+                    categories
+                    with parents.<br>
+                    This will allow users to navigate to a PLP of third level category.
+                </p>
+            </div>
+
+            <div>
+                <label class="flex mt-2 items-center">
+                    <input v-model="context.showSoldOutProducts"
+                           class="accent-brand-500 mr-3 h-5 w-5"
+                           type="checkbox">
+                    Show <span class="font-medium px-1">Sold out</span> in search and recommendations</label>
+
+                <p class="text-gray-400">
+                    Products with a SoldOut datakey are by default hidden in search and recommendations.
```

Comment:
Bør vi ikke by default vise alle produkter ligesom det er i dag? 
På den måde er der ikke en ændring i funktionalitet for alle andre dataset.